File: dr-quine/C/colleen.c
#include <stdio.h>
void	function() {}
/*
	comment outside of main
*/
int	main()
{
	/*
		comment in main
	*/
	char *s = "#include <stdio.h>%cvoid	function() {}%c/*%c	comment outside of main%c*/%cint	main()%c{%c	/*%c		comment in main%c	*/%c	char *s = %c%s%c;%c	function();%c	printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10);%c}%c";
	function();
	printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10);
}

File: RUST/Colleen.rs
fn	print()
{
	let	s = "fn\tprint()\n{{\n\tlet\ts = {0:?};\n\tprintln!({0:?}, s);\n}}\n/*\n *\tComment outside of main\n */\nfn\tmain()\n{{\n\t/*\n\t *\tComment in main\n\t */\n\tprint();\n}}";
	println!("fn\tprint()\n{{\n\tlet\ts = {0:?};\n\tprintln!({0:?}, s);\n}}\n/*\n *\tComment outside of main\n */\nfn\tmain()\n{{\n\t/*\n\t *\tComment in main\n\t */\n\tprint();\n}}", s);
}
/*
 *	Comment outside of main
 */
fn	main()
{
	/*
	 *	Comment in main
	 */
	print();
}
